gui/src/components/Backups.vue

```diff
@@ -0,0 +1,130 @@
+<template>
+    <div>
```

Comment:
![Screen Shot 2020-06-30 at 9 23 29](https://user-images.githubusercontent.com/1052473/86068884-5ff49c00-bab3-11ea-83e0-56bb280ba8c7.png)
It is now left aligned when you make a browser window bigger, but I think this should be centered.